src/Commands/sql/sanitize/SanitizeUserTableCommands.php

```diff
@@ -80,6 +84,20 @@ public function sanitize($result, CommandData $commandData): void
             $messages[] = dt('User emails sanitized.');
         }
 
+        // Sanitize username.
+        if ($this->isEnabled($options['sanitize-username'])) {
+            [$name_table, $name_column] = $this->getFieldTableDetails('user', 'name');
+            [$uid_table, $uid_column] = $this->getFieldTableDetails('user', 'uid');
+            assert($uid_table === $name_table);
+
+            // Updates usernames to the pattern user_%uid.
+            $query
+                ->condition($uid_column, 0, '>')
+                ->expression($name_column, "CONCAT('user_', $uid_column)");
```

Comment:
Looking at the email code, it appears we'd need a different format for different DB drivers I think?